docs/token-guides/wsteth-bridging-guide.md

```diff
@@ -0,0 +1,353 @@
+# wstETH rollup bridging guide
+
+:::warning Disclaimer
+This is an unofficial and incomplete version of the bridging guide intended for informational purposes only and does not state any commitment from the Lido DAO or from the Network Expansion Workgroup.
+:::
+
+## Intro
+
+This document is intended for the developers representing network/rollup foundations and DAOs looking to bridge Lido's wstETH on Ethereum L2 (rollup) networks.
+
+:::info
+This guide doesn't cover yet bridging of rebasable stETH token nor bridging to non-L2-rollup networks. However, please note that bridging stETH in a standardish way might cause a loss of user assets due to the rewards accrued being stuck on an L1 bridge.
+:::
+
+While technically, it is feasible to bridge the wstETH token on an L2 network as any other standard plain ERC-20 compatible token, it might not be aligned with the long-term vision of the Lido DAO for stETH future-proof adoption and general community sentiment.
+
+This guide covers the recommendations as well as provides general guidelines, and reveals the logic behind to smooth the process. It's essential to understand that conforming to or diverging from these guidelines won't ensure the recognition or rejection of a specific proposal by the Lido DAO. Nonetheless, adhering to these guidelines substantially increases the likelihood of gaining support from the Network Expansion Workgroup (NEW) and community. Ultimately, the final decision is determined by the outcome of the voting process.
+
+:::info
+Please send any of your feedback on the guide to the NEW — the doc gets iterative updates.
+:::
+
+## Why is this guide needed
+
+As said before, the "standard" way to bridge an ERC-20 token is to deploy on L2 a non-upgradable token and use the general bridge contract, this guide proposes to implement a more complex solution.
+
+In short, to deploy upgradable custom (still native) bridges on L1 and L2 and an upgradable token on L2, all governed by the Lido DAO on L1 ([Aragon Agent contract](https://etherscan.io/address/0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c)) via a dedicated [AAVE-like governance executor](https://github.com/lidofinance/governance-crosschain-bridges/) contract on L2. This architecture is proposed to provide the following capabilities.
+
+1. Have the ability to pass additional data and revamp logic. Specifically for the bridging rebasable stETH (requires passing wstETH/stETH rate). The stETH token is not a general-purpose token but an asset built on top of the liquid staking protocol, which has moving and upgradable parts.
+2. Let the token be "future proof" and avoid the potential need to do high-cost liquidity migration as Ethereum continues to evolve and new standards like ERC-2612/ERC-1271 get added.
```

Comment:
Future-proofing the token, for example, to avoid high-cost liquidity migration as Ethereum continues evolving and new standards like ERC-2612/ERC-1271 are being adopted.

P.S. Probably we can merge this point with revamping token logic.
P.P.S. Consider adding links to these ERCs.